x-pack/plugins/secops/public/components/page/hosts/hosts_table/index.tsx

```diff
@@ -115,22 +114,25 @@ const getHostsColumns = () => [
             dataProvider={{
               and: [],
               enabled: true,
+              excluded: false,
               id: node._id!,
               name: hostName,
-              negated: false,
-              queryMatch: `host.id: "${escapeQueryValue(node.host!.id!)}"`,
-              queryDate: `@timestamp >= ${moment(
-                node.firstSeen!
-              ).valueOf()} and @timestamp <= ${moment().valueOf()}`,
+              kqlQuery: '',
+              queryMatch: {
+                displayField: 'host.name',
+                displayValue: hostName,
+                field: 'host.id',
+                value: node.host!.id!,
```

Comment:
Would the following be an accurate description of this `queryMatch`?
> When a user drags this `dataProvider` to the timeline, it will appear to the user that the query is based on `host.name` (based on the contents of the card shown in the timeline query builder), but when the query executes, it will actually match on `host.id`).

Reply:
The user is dragging `host.name` but when we want to make sure that they are dragging exactly what they pick but the query need to be done with `host.id` to be exact since you can have the same name of host multiple time.